src/app/codegen-data-model-provider/CodegenDataModelProvider.cpp

```diff
@@ -14,6 +14,8 @@
  *    See the License for the specific language governing permissions and
  *    limitations under the License.
  */
+#include "app/data-model-provider/Provider.h"
```

Comment:
Why is this in quotes, not angle brackets like our other includes?  And why is it here at all?  I'd think CodegenDataModelProvider.h includes that header, and has to....